suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -180,6 +179,21 @@ export const useAddCoinAccount = () => {
         setNetworkWithTypeToBeAdded([network, defaultType]);
     };
 
+    const accountsForNetworSymbolkWithAccountType = ({
```

Comment:
```suggestion
    const getAccountsForNetworSymbolkWithAccountType = ({
```

Reply:
Why don't you use `selectDeviceAccountsForNetworkSymbolAndAccountType`?

Reply:
584c12fa8a